surfactant/infoextractors/docker_file.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC
+# See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: MIT
+import gzip
+import json
+import subprocess
+import tempfile
+
+import surfactant.plugin
+from surfactant.sbomtypes import SBOM, Software
+
+
+def supports_file(filetype: str) -> bool:
+    return filetype in ("Docker archive tar", "Docker archive gzip")
+
```

Comment:
```suggestion
def is_docker_scout_installed():
    # Check that Docker Scout can be run unless it's disabled
    result = subprocess.run(["docker", "scout", "--help"], capture_output=True, check=False)
    if result.returncode != 0:
        logger.warning(
            "Install Docker Scout to scan containers for additional information"
        )

# Check if Docker Scout is installed when this Python module gets loaded
disable_docker_scout = not is_docker_scout_installed()
```

The above should result in the check for Docker Scout happening just once when this module gets loaded. Another idea that came to mind is adding a "check_plugin_prereqs" hookspec that would let plugins that have choose to implement it do some checks when they first get loaded. That might be a more elegant solution, coupled with a future user settings for plugins feature.

I think in terms of UX the lack of Docker Scout shouldn't be a hard failure that requires disabling -- there are likely to be plenty of samples people analyze that don't have any Docker images. Ideally we'd know ahead of time that one of the files they'll be analyzing is a Docker image, and then warn them... a more realistic alternative to omniscience could be printing the warning in extract_docker_info() if/when a Docker file is encountered.